test/test_optim.py

```diff
@@ -4119,19 +4143,24 @@ def avg_fn(p_avg, p, n_avg):
         for b_avg, b_swa in zip(dnn.buffers(), averaged_dnn.module.buffers()):
             self.assertEqual(b_avg, b_swa)
 
-    def test_averaged_model_exponential_buffers(self):
+    @parametrize("use_multi_avg_fn", [True, False])
+    def test_averaged_model_exponential_buffers(self, use_multi_avg_fn):
```

Comment:
This test also looks very similar to test_averaged_model_exponential, could they abstract out the common areas?

Reply:
Done

Reply:
Merged them